spec/fixtures/generate_external_certs.sh

```diff
@@ -0,0 +1,322 @@
+#!/bin/bash
+
+# This script re-generates all of the keys and certs within the external_certs directory.
+# The files there are used by the spec/features/settings/external_cert_feature_spec test
+
+function init() {
```

Comment:
Function declarations in bash/ksh/zsh should be either `function fname {` or `fname() {`; combining the two works in horrible-language bash, but will fail in other shells.  This is a bash script, so it doesn't currently matter, but is worth bearing in mind for general shell scripting purposes.

Reply:
But that's how it looks in Javascript. Just kidding. I'll change them.

Reply:
Have dropped the 'function' at the start of each function.